_clanky/výběr-katalogizačního-systému-pro-město-Brno.md

```diff
@@ -0,0 +1,83 @@
+
+Co dodat na závěr? Výběr vhodného katalogizačního systému je náročnou disciplínou. Častokrát je těžké si správně vybrat bez předešlých zkušeností a detailní znalosti problematiky  Někdy se stane, že si samospráva (jako například v případě Brna) napoprvé nevybere pro svoje potřeby nejvhodnější technologii. Tímto samozřejmě **netvrdíme**, že CKAN nebo jiný systém je horší než systém HUB. Při výběru je totižto nutné vzít v potaz technologická, personální nebo finanční specifika každého města a ta pak určují, jaký systém je nejvhodnější. ArcGIS HUB například **určitě není vhodným řešením** pro organizace, které nemají podstatnou část svých dat ve službách nebo geodatabázích postavených nad architekturou od ESRI.
+
+Pro Brno však bylo toto řešení nejlepší z možných, neboť funkcionality a pokročilé technologické řešení popsané výše umožňují poskytovat odborné veřejnosti otevřená data bez nutnosti vynaložení velkých finančních prostředků. Kromě Brna, jako jediného většího města v ČR, využívá v současnosti ArcGIS HUB pro publikaci open dat ještě [Agentura ochrany přírody a krajiny][AOPK]. V zahraničí je užívání ArcGIS HUBu mnohem rozšířenější - využívají jej například města jako [Sydney][SYDNEY], [Washington D.C.][WASHINGTON] nebo [Pařížská metropolitní oblast][PARIS]. 
+
+Pokud Vás zajímá problematika správy otevřených dat v kontextu města Brna ještě podrobněji, neváhejte se na Brno obrátit se svým dotazem, a to na e-mail data@brno.cz.
```

Comment:
Ten email má být odkaz na mailtu:data@brno.cz

Reply:
majl som tam dal ako odkaz tak ako sú ostatné linky ale nie som si istý či je to správne... ak nie tak potom tak ako si to tu napísal?